documentation/docs/25-build-and-deploy/60-adapter-cloudflare.md

```diff
@@ -28,14 +28,19 @@ export default {
 			routes: {
 				include: ['/*'],
 				exclude: ['<all>']
-			}
+			},
+			external: [ 'fs' ]
 		})
 	}
 };
 ```
 
 ## Options
 
+The adapter accepts two options: `routes` and `external`.
+
+### Option routes
```

Comment:
```suggestion
### `routes`
```

Reply:
This was marked as resolved without making the change.

Reply:
Good catch!